tests/kafkatest/sanity_checks/test_verifiable_producer.py

```diff
@@ -36,21 +36,25 @@ def __init__(self, test_context):
                                   topics={self.topic: {"partitions": 1, "replication-factor": 1}})
 
         self.num_messages = 100
-        # This will produce to source kafka cluster
-        self.producer = VerifiableProducer(test_context, num_nodes=1, kafka=self.kafka, topic=self.topic,
-                                           max_messages=self.num_messages, throughput=1000)
 
     def setUp(self):
         self.zk.start()
-        self.kafka.start()
 
-    @parametrize(producer_version=str(LATEST_0_8_2))
-    @parametrize(producer_version=str(TRUNK))
-    def test_simple_run(self, producer_version=TRUNK):
+
+    @parametrize(producer_version=str(LATEST_0_8_2), compression_type=None)
+    @parametrize(producer_version=str(TRUNK), compression_type=None)
+    @parametrize(producer_version=str(TRUNK), compression_type=["snappy"])
```

Comment:
The reasoning is that this is a smoke test, and compression I guess is one of the basic behaviors. I don't have strong opinion about this. @granders Do you think it makes sense to do smoke test for compression if we have a separate produce -> consume -> verify system tests for compressed topics?